File: pinyougou-interface/src/main/java/cn/itcast/core/service/message/MessageService.java
package cn.itcast.core.service.message;

import cn.itcast.core.pojo.user.User;

import java.text.ParseException;

public interface MessageService {
    /**
     * 修改用户的基本信息
     * @param user
     * @return
     */
    void update(User user) throws ParseException;

    /**
     * 查询回显用户逇基本信息
     * @return
     */
    User findByOne(String name) throws ParseException;
}

File: pinyougou-web-user/src/main/webapp/js/service/messageService.js
//服务层
app.service('messageService',function($http){
	//发送用户信息
	this.update=function(entity){
		return $http.post('/message/update.do?user='+user);
	}

	//查询回显用户信息
	this.findByOne=function(){
		return $http.post('/message/findByOne');
	}
	
});